content/rs/administering/designing-production/supported-platforms.md

```diff
@@ -13,7 +13,7 @@ Note: This document covers RS 5.0 and up.
 
 | **Platform** | **Versions/Information** |
 |------------|-----------------|
-| Ubuntu |  14.04, 16.04 64-bit, Server or Desktop. The Server version is recommended for production deployments. |
+| Ubuntu (Server or Desktop) |  14.04, 16.04 64-bit, 18.04 64-bit. The Server version is recommended for production deployments. |
```

Comment:
1. Move to the last sentence
2. All are 64 bit versions

Reply:
Check it out at: https://docs.redislabs.com/staging/5.4-ubuntu-18.04/rs/administering/designing-production/supported-platforms/